libs/shared/flagd-core/src/lib/targeting/fractional.ts

```diff
@@ -0,0 +1,97 @@
+import {flagdPropertyKey, flagKeyPropertyKey, targetingPropertyKey} from "./common";
+import MurmurHash3 from "imurmurhash";
+
+export const fractionalRule = "fractional";
+
+export function fractional(data: unknown, context: Record<any, any>): string | null {
```

Comment:
Can the record be `Record<unknown, unknown>`? If not, can you add a comment to have the linter ignore this?

Reply:
No this is not possible as `Record` expect any type. I disabled the lint check on this line